metricbeat/module/vsphere/datastore/datastore_test.go

```diff
@@ -27,7 +27,7 @@ import (
 )
 
 func TestFetchEventContents(t *testing.T) {
-	model := simulator.ESX()
+	model := simulator.VPX()
```

Comment:
Can we please add the comment her, on why we prefer using VPX to ESX in datastore